docs/changelog/82409.yaml

```diff
@@ -0,0 +1,19 @@
+pr: 82409
+issues:
+  - 81210
+  - 52728
+area: Search
+type: enhancement
+summary: Allow doc-values only search on number types
+highlight:
+  notable: false
+  title: Allow doc-values only search on number types
+  body: |-
+    Allows searching on number field types (long, short, int, float, double,
```

Comment:
Can you replace this text with the following:

> Doc-value-only fields allow users to trade slower query performance for an increased indexing rate and storage space efficiency. This is achieved by disabling inverted structures such as the inverted index or points, and using the columnar storage format of doc values to run queries instead, while still providing the same usual great performance for aggregating and sorting data. Elasticsearch 8.1.0 supports term and range queries on numeric, date, keyword, ip, and boolean doc-value-only field types.

Reply:
Thanks @ywelsch 👍